src/components/NcRichText/NcRichText.vue

```diff
@@ -495,10 +528,11 @@ export default {
 							id,
 							disabled: !this.interactive,
 							'onUpdate:modelValue': (value) => {
-								this.$emit('interact:todo', { id, label: labelParts.join(''), value })
+								this.$emit('interact:todo', id)
 							},
 						}, labelParts)
-						return h(type, props, [inputComponent])
+
+						return h(tag, attrs, [inputComponent, nestedNode])
```

Comment:
should be `type` instead of `tag` and `props` instead of `attrs`. Try to check Vue 3 docs on h function and play with it in styleguide

Reply:
Yeah, I mistakenly overwrote it